### ISSUE TITLE
feat: add the ability to the extension to call the ignite commands

### ISSUE THREAD

**Issue by Pantani** [ignite/apps#117]
Add the ability to the extension to call the ignite commands

**Comment by julienrbrt** [comment]
That's already possible like this: https://github.com/ignite/apps/blob/e5cd28a/appregistry/cmd/install.go#L34-L39
Not the best I admit but does the job.
What are you thinking about?

**Comment by Pantani** [comment]
I was doing some tests trying to call the commands through GRPC, but in the end, it was the same solution with overengineer.

https://github.com/ignite/cli/pull/4252

WDYT?

**Comment by julienrbrt** [comment]
I look actually pretty handy, but it is indeed less flexible than just importing the cobra command. We would need a way to get stderr, stdout from that API.
However me may need to use this if we don't want to depend on cobra in plugins.

**Comment by Pantani** [comment]
The trick to importing the Cobra command is the ignite version. The version from the terminal can be different from the version from the app using an old building. However, using thought GRPC can break the app if the commands change.

**Comment by julienrbrt** [comment]
Right, 100% agree, adding calling it via grpc is great for that, however we may want the api to do more (outputs, checking version indeed)

**Comment by Pantani** [comment]
Now we are calling the Ignite cobra command,  but we can also make this through GRPC

https://github.com/ignite/apps/blob/f0f1165ca3f1c0a45bf5c182af6447309696e896/spaceship/cmd/chain.go#L251-L255
